api/src/main/scala/org/ensime/api/config.scala

```diff
@@ -25,6 +25,8 @@ case class EnsimeConfig(
   def root = rootDir
   def debugVMArgs = debugArgs
   def referenceSourceJars = referenceSourceRoots
+  def disableSourceMonitoring = true
```

Comment:
Btw I'm thinking undocumented flags for use in the short term for very big projects.
It's a shame, but class monitoring is useless in my situation anyway, because I don't have continuous build. Source monitoring less so, but I can live without it.

Obviously looking forward to getting rid of these : )